resources/views/components/remove-member-modal.blade.php

```diff
@@ -0,0 +1,30 @@
+<div x-data="modal()">
+    <button class="secondary" type="button" @click="showModal">{{ __('Remove') }}</button>
```

Comment:
This should read "Leave" if the member is the current user.

Reply:
Should we also update the modal title to something like "Leave from your organization"?